ui/app/AppMain.qml

```diff
@@ -146,9 +146,9 @@ RowLayout {
         case Constants.chat: sectionId = 0; break;
         case Constants.wallet: sectionId = 1; break;
         case Constants.browser: sectionId = 2; break;
-        case Constants.profile: sectionId = 3; break;
-        case Constants.node: sectionId = 4; break;
-        case Constants.ui: sectionId = 5; break;
+        case Constants.profile: sectionId = 4; break;
+        case Constants.node: sectionId = 5; break;
+        case Constants.ui: sectionId = 6; break;
```

Comment:
It's not like we're changing these too often, but this is actually much better solved with an enum.
Especially because this function doesn't honour sections that have been disabled.